pkg/eventhandler/eventhandlers_test.go

```diff
@@ -142,7 +155,7 @@ func (mei *matcherErrorIs) String() string {
 	return fmt.Sprintf("%#v", mei.targetErr)
 }
 
-func TestErrorGettingJobConfig(t *testing.T) {
+/*func TestErrorGettingJobConfig(t *testing.T) {
```

Comment:
Is there any reason why this test is commented out?

Reply:
It seems like I forgot to rewrite the tests to use the `go-sdk`. It should be added now.